### PR TITLE
dotCMS/core#16767 QA Feedback 1

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/content/components/dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.component.ts [hunk 1]
```diff
@@ -77,12 +84,13 @@ export class DotEditPageViewAsControllerComponent implements OnInit {
      * @param {DotPersona} { keyTag }
      * @memberof DotEditPageViewAsControllerComponent
      */
-    deletePersonalization({ keyTag }: DotPersona): void {
+    deletePersonalization({ keyTag, name }: DotPersona): void {
```
Comment (alfredo-dotcms) [inline src/app/portlets/dot-edit-page/content/components/dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.component.ts]: add `name` to doc

File: src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ export class DotPersonaSelectorOptionComponent implements OnInit {
                             messages['modes.persona.personalized']
                     )
                 );
-        }, 0);
+        }, 10);
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.ts]: why?

File: src/app/portlets/dot-edit-page/content/components/dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.component.ts [hunk 3]
```diff
@@ -34,7 +37,11 @@ export class DotEditPageViewAsControllerComponent implements OnInit {
     ngOnInit(): void {
         this.isEnterpriseLicense$ = this.dotLicenseService.isEnterprise();
         this.previewDeviceLabel$ = this.dotMessageService
-            .getMessages(['editpage.viewas.previewing'])
+            .getMessages([
+                'editpage.viewas.previewing',
+                'editpage.personalization.delete.confirm.header',
```
Comment (alfredo-dotcms) [inline src/app/portlets/dot-edit-page/content/components/dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.component.ts]: update tests?

File: src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts [hunk 4]
```diff
@@ -41,6 +44,7 @@ export class DotPersonaSelectorComponent implements OnInit, OnChanges {
     @ViewChild('searchableDropdown')
     searchableDropdown: SearchableDropdownComponent;
 
+    currentOffset = 0;
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts]: is this what solves `Bryan's issue` ?

File: src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts [hunk 5]
```diff
@@ -50,18 +54,20 @@ export class DotPersonaSelectorComponent implements OnInit, OnChanges {
     constructor(public paginationService: PaginatorService) {}
 
     ngOnChanges(changes: SimpleChanges): void {
-        if (!changes.value.firstChange) {
-            this.getPersonasList();
+        this.paginationService.url = `v1/page/${this.pageId}/personas`;
+
+        if (
+            this.isPersonalizeStateUpdated(changes.value.previousValue, changes.value.currentValue)
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts]: update tests for this?

File: src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.ts (hunk 2 above)
Comment (fmontes) [reply src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.ts]: Weird things with the change detector when you're getting the message key in a component that your're passing as template.

File: src/app/portlets/dot-edit-page/content/components/dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.component.ts (hunk 3 above)
Comment (fmontes) [reply src/app/portlets/dot-edit-page/content/components/dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.component.ts]: Don't have tests yet.

File: src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts (hunk 4 above)
Comment (fmontes) [reply src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts]: Removing, what a first approach I was trying.

File: src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts (hunk 5 above)
Comment (fmontes) [reply src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts]: No tests yet for all this changes.

**Pull request merged**